src/workflows/dags/airqo_measurements.py

```
@@ -694,13 +667,57 @@ def extract_devices_missing_calibrated_data(**kwargs) -> Tuple[pd.DataFrame, str
+            data=data,
+            device_category=DeviceCategory.LOWCOST,
+            data_type=DataType.RAW,
+            frequency=Frequency.HOURLY,
+        )
+
+    @task()
+    def aggregate(data: pd.DataFrame):
+        return DataUtils.aggregate_low_cost_sensors_data(data=data)
+
+    @task(provide_context=True, retries=3, retry_delay=timedelta(minutes=5))
+    def extract_hourly_weather_data(start_date: str, end_date: str, **kwargs):
```

Ruff (0.14.1)

Unused function argument: `kwargs`

(ARG001)